src/Microsoft.TestPlatform.CommunicationUtilities/TestRequestSender.cs

```diff
@@ -23,6 +24,10 @@ public class TestRequestSender : ITestRequestSender
         // Time to wait for test host exit
         private const int ClientProcessExitWaitTimeout = 10 * 1000;
 
+        // The minimuim protocol version a testhost has to implement to be aware it should ask
+        // VS to attach the debugger to the said testhost.
+        private const int MinimumProtocolVersionWithDebugSupport = 3;
```

Comment:
>MinimumProtocolVersionWithDebugSupport [](start = 26, length = 38)

Can we put this in one central place? Seems to be used in other places too.

Reply:
Done, defined a constant in ObjectModel. We should be safe on the compat side with this one.